docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst

```diff
@@ -8,6 +8,41 @@ Tracheal, Bronchus, and Lung Cancer
    :local:
    :depth: 1
 
+.. list-table:: Abbreviations
```

Comment:
Good addition!  Let's add this to our template.

Reply:
Just made a PR!